packages/contracts-bedrock/test/vendor/eas/SchemaRegistry.t.sol

```diff
@@ -0,0 +1,290 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import { Test } from "forge-std/Test.sol";
+import { console } from "forge-std/console.sol";
+import { SchemaRegistry, SchemaRecord } from "src/vendor/eas/SchemaRegistry.sol";
+import { ISchemaResolver } from "src/vendor/eas/resolver/ISchemaResolver.sol";
+
+contract SchemaRegistryTest is Test {
```

Comment:
You should do `contract SchemaRegistryTest is CommonTest` and then update the following code: https://github.com/ethereum-optimism/optimism/blob/a47441c8995e1b02fcddbbe558c65be3763a8a7f/packages/contracts-bedrock/test/setup/Setup.sol#L112

so that `schemaRegistry` and `eas` are defined and set in the `Setup` file. Then they will be available globally here, so you don't need to deploy the `registry`. This tests the L2 genesis script, so the same setup that creates an L2 genesis for production is tested in the unit test. The other unit tests do this